static/stable/web_help/docs/zowe_profiles_list_base-profiles.html

```diff
@@ -15,7 +15,7 @@ <h4>Options</h4>
 <li>
 <p><code>--show-contents</code>  | <code>--sc</code> <em>(boolean)</em></p>
 <ul>
-<li>List base profiles and their contents. All profile details will be printed as
+<li>List base  profiles  and their contents. All profile details will be printed as
```

Comment:
not sure why those spaces are added there. Everything seems to have looked ok even prior to this change

Reply:
The addition of extra spaces is a bug we have noticed recently in Zowe CLI. For the web help, it should be fine because I believe multiple spaces in HTML get rendered as just one.